src/styles/themes/b2c/variables.scss

```diff
@@ -18,38 +17,23 @@ $logo-mobile-height: var(--logo-mobile-height);
 // General spaces
 $space-default: 15px;
 
-// FONTS
+// Fonts
 $font-family-regular: var(--font-family-regular);
 $font-family-bold: var(--font-family-bold);
 $font-family-condensedregular: var(--font-family-condensedregular);
 $font-family-condensedbold: var(--font-family-condensedbold);
 
-// FONT SIZES
+// Font sizes
 $font-size-corporate: 0.875rem; // 14px is the bootstrap 3 default font size (1rem should be around 16px)
 
-//CORPORATE DESIGN COLORS
-
+// Corporate design colors
 $CORPORATE-PRIMARY: var(--corporate-primary);
```

Comment:
the color definitions look a bit strange, maybe a hint where the actual implementation can be found (properties.scss?) would be helpful